app/models.py

```diff
@@ -3648,3 +3660,42 @@ def create(cls, **kwargs):
         code = secrets.token_urlsafe(32)
 
         return super().create(code=code, **kwargs)
+
+
+class SyncEvent(Base, ModelMixin):
+    """This model holds the events that need to be sent to the webhook"""
+
+    __tablename__ = "sync_event"
+    content = sa.Column(sa.LargeBinary, unique=False, nullable=False)
+    taken_time = sa.Column(ArrowType, default=None, nullable=True, server_default=None)
```

Comment:
We'll need an index by taken time and create time so that the deadletter uses it

Reply:
Added indexes